video/out/vo_kitty.c

```diff
@@ -382,6 +387,14 @@ static int preinit(struct vo *vo)
     }
 #endif
 
+    if (p->opts.auto_multiplexer_passthrough) {
+        if (getenv("TMUX")) {
+            p->dcs_prefix = DCS_TMUX_PREFIX;
+        } else if (!strcmp(getenv("TERM"), "screen")) {
```

Comment:
Don't feed result of `getenv` to `strcmp` directly as it may be NULL.

Reply:
Changed to check for `STY` environment variable instead

Reply:
> Changed to check for `STY` environment variable instead

I recommend squashing the fix on top of the earlier commit. Best not to merge buggy commit to begin with, even if fixed in a commit immediately after.

Reply:
fixed commit history